### ISSUE TITLE
Can't view second page when filter results is more than one page

### ISSUE THREAD

**Issue by code2expert** [aspnet/Docs#11056]
When my filter returns more than one page, the page goes back to the full list when I click on next. Please help update the tutorial to correct this.

Thanks for this very interesting learning experience. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 877f0457-37d9-9a34-e2ce-beb41d717086
* Version Independent ID: e9b0d4d1-e98c-fb96-415f-0ff4e37d3658
* Content: [Razor Pages with EF Core in ASP.NET Core - Sort, Filter, Paging - 3 of 8](https://docs.microsoft.com/en-us/aspnet/core/data/ef-rp/sort-filter-page?view=aspnetcore-2.2)
* Content Source: [aspnetcore/data/ef-rp/sort-filter-page.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/data/ef-rp/sort-filter-page.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
We'll look at this on the next doc sweep. This is not production code.

**Comment by code2expert** [comment]
Hi Rick,

Thanks for your feedback. However, I found a way around the issues by setting a condition on the pageSize. I set the pageSize before checking to see if the search string is not empty, then set the pageSize if the search string returns a result. I practically set the page size to the count of the records in the retuned resultset from the searchstring.

I’m still looking at doing this writhing the search results with the set pageSize but the solution above works just fine too.

Thanks
